lib/hca/enrollment_system.rb

```diff
@@ -840,9 +840,9 @@ def remove_ctrl_chars!(value)
 
     def get_va_format(content_type)
       # ES only accepts these strings for 'va:format': PDF,WORD,JPG,RTF
-      extension = MIME::Types[content_type].first.extensions.first
+      extension = MIME::Types[content_type]&.first&.extensions&.first
 
```

Comment:
I wonder if instead of defaulting to `PDF` here for an invalid mime type, we should at the very least log the invalid type or something similar?

Reply:
My main thought with logging was to log when the mime type is something that is invalid. But I guess I do see we log that here. Maybe a statsD metric? My main concern was to see how often this even happens, although it really shouldn't since we have that validation on the attachments controller. It seems weird that we just default anything other than a `WORD`, `JPG`, or `RTF` to a `PDF`.

Reply:
Yeah, that makes sense to me. Didn't mean to derail this PR too much. At some point I'd love to look into our `enrollment_system.rb` and see about making it less confusing, but that's a future thing to deal with.